### ISSUE TITLE
[arabic_izza] A key is missing from the Phone numeric layer that would enable the return to default layer.  

### ISSUE THREAD

**Issue by MaleBou** [keymanapp/keyboards#324]

**Comment by MaleBou** [comment]
There are also four Keys ("، |، {، }) on numeric layer not working because of small letters used instead of capital ones to identify them.

**Comment by MaleBou** [comment]
The Phone default layer could be made almost perfect by swapping some keys on the third line to look similar to the ones on the physical keyboard, and put letter ة with ه, and the comma in place of ة.

**Comment by darcywong00** [comment]
We've created a new guide which we hope simplifies the keyboard submission process.
https://help.keyman.com/developer/keyboards/starter/step-1.php

It involves a template repository [keyboards-starter](https://github.com/keymanapp/keyboards-starter)

I recommend following this guide for submitting your updates to the touch layout. For your convenience, I'm attaching a zip of the current source to the `arabic_izza` keyboard that got merged last week. 
[arabic_izza.zip](https://github.com/keymanapp/keyboards/files/2073072/arabic_izza.zip)


When you get to step 3 of the guide, you'll unzip the folder in your copy of *keyboards-starter* so that your local directory has something like
```
c:\projects\keyboards-starter\arabic_izza\release\a\arabic_izza\...
```

With Keyman Developer 10, you'll then apply your updates to 
```
c:\projects\keyboards-starter\arabic_izza\release\a\arabic_izza\source\arabaic_izza-layout.js
```

**Comment by MaleBou** [comment]
Does PR #325 need to stay open? all the revisions have been done and everything seems to work perfectly. Just small improvements to the phone layout are needed. putting only 8 keys on the default layer has improved enormously the clarity and typing speed. I think they should be kept as they are.

**Comment by darcywong00** [comment]
> Does PR #325 need to stay open? seem to be conflicting with PR #330.

Sorry, I won't be available to provide any assistance for #325. I think closing #325 and continuing your work on #330 is best.

Somebody will need to resolve the merge conflicts. It's a complication from modifying an existing keyboard via the *keyboards-starter* repo.

**Comment by MaleBou** [comment]
This is closed for now because a new PR was created here #342 .
